src/ansys/fluent/core/workflow.py

```diff
@@ -1121,6 +1169,14 @@ def __dir__(self):
                 )
             )
         )
+        unwanted = [
+            "reset_workflow",
+            "initialize_workflow",
+            "load_workflow",
+            "create_new_workflow",
+        ]
+        dir_list = [e for e in dir_list if e not in unwanted]
```

Comment:
actually both args could be `set`s, leading to a bit of a nice rewrite over all here